components/events/EventMultiSelect.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <cv-multi-select
+    class="event-multi-select"
+    :theme="theme"
+    :label="label"
+    :options="options"
+    :selection-feedback="feedback"
+    @change="selectedOptions => updateWholeFilter(filterType, selectedOptions)"
```

Comment:
This action is related to the events page, so it should go on the parent component. On [this article](https://michaelnthiessen.com/pass-function-as-prop/) you can read how to pass functions on Vue: The child component (`AppMultiSelect`) emits an event and the parent (when you use `<AppMultiSelect>`) is listening to it and acts when it receives the event. So, here, to emit the event, you can do: 
```
@change="$emit('change-on-multi-select', $event)"
```
And on the parent you are listening to the event and acting:
```
@change-on-multi-select="updateWholeFilter(regionsFilters, $event)"
```